test/Java8andUp/playlist.xml

```diff
@@ -1532,5 +1532,25 @@
 			<subset>SE90</subset>
 		</subsets>
 	</test>
+	<test>
+		<featureIds>
+			<featureId>134091</featureId>
```

Comment:
Is this ID an internal bug #?  If so, please replace it with an id that's meaningful in the open

Reply:
Okay. Talking to @renfeiw about what this should be. There are many other similar <featureIds> in the test code.